protolator/dynamic_test.go

```diff
@@ -11,10 +11,13 @@ import (
 	"testing"
 
 	"github.com/hyperledger/fabric-config/protolator/testprotos"
-	"github.com/stretchr/testify/assert"
+
+	. "github.com/onsi/gomega"
```

Comment:
We don't really have a consistent approach to import grouping, did you make a conscious decision to separate the dot imports from the others or is it an IDE snafu?

Reply:
I tend to separate dot imports as a personal preference. My go imports plugin doesn't seem to care where it's grouped with since it doesn't automatically collapse it. If we want to follow some convention I'm fine with that as well.